include/NAS2D/Filesystem.h

```diff
@@ -50,16 +57,64 @@ class Filesystem
 
 	void toggleVerbose() const;
 
+    void setWorkingDirectory(const std::filesystem::path& p);
+    std::filesystem::path getWorkingDirectory();
+
 private:
-	Filesystem(const Filesystem&) = delete;
-	Filesystem& operator= (const Filesystem&) = delete;
 
-	bool closeFile(void *file) const;
-	const char* getLastPhysfsError() const;
+    std::filesystem::path getExePath() const;
```

Comment:
"EXE" is a Windows term for a binary -- I think most programmers/users would understand exactly what it means but perhaps there's a better name for this? Executable? Binary? App?

This is only a minor thing, really and is up for discussion, it can be changed in later versions if needed.

Reply:
I like Binary, but concede to future discussion or consensus of the group. A preliminary googling of "What are executables called in Linux?" and "What are executables called in MacOSX?" returns ".app" for Mac and "Whatever the hell you want because Linux uses the file permissions to determine executability and with what program." in Linux.